pom.xml

```diff
@@ -64,8 +64,8 @@
       </dependency>
       <dependency>
         <groupId>org.daisy.pipeline.modules.braille</groupId>
-        <artifactId>braille-modules-parent</artifactId>
-        <version>1.9</version>
+        <artifactId>braille-modules-bom</artifactId>
+        <version>1.9.1-SNAPSHOT</version>
```

Comment:
@rdeltour It seems the maven-release-plugin didn't detect this SNAPSHOT dependency :(.